schema2Markdown.js

```diff
@@ -0,0 +1,255 @@
+const fse = require('fs-extra');
+const yaml = require('js-yaml');
+const path = require('path');
+
+function processProperty(propertyName, propertyDetails, schemaExamples) {
+    let markdownContent = '';
+
+    if (propertyName === 'type') {
+        markdownContent += `### Type\n\n`;
+        markdownContent += `\`${propertyDetails.const}\`\n\n`;
+        return markdownContent;
+    }   
```

Comment:
As its so important, we should move the type property up out of properties. Hence, just skip it here:

```suggestion
    if (propertyName === 'type') {
        //skip rendering the type property as it should be rendered at the top level
        return markdownContent;
    }   
```